### ISSUE TITLE
Accessibility:List: The “selected” state of list items is not read immediately after selecting the list items. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3479]
Description: The “selected” state of list items is not read immediately after selecting the list items. 
Also, the “selected” state of list items is not read, even while navigating between the list items using arrow keys. 

Expected: The state ‘selected’ should be read out, after selecting the list items, also when list items are accessed. 

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94647942-2aa85600-030f-11eb-843e-f88530e365ac.png)

![image](https://user-images.githubusercontent.com/32538291/94647954-3005a080-030f-11eb-81cd-abd732e961de.png)

![image](https://user-images.githubusercontent.com/32538291/94647958-33992780-030f-11eb-9a33-672704e382e6.png)
